perun-utils/ldapc-scripts/schemas/perun-schema.ldif

```diff
@@ -44,12 +44,21 @@ olcAttributeTypes: {39}( 1.3.6.1.4.1.8057.2.80.45 NAME 'adminOfVo' DESC 'The id
 olcAttributeTypes: {40}( 1.3.6.1.4.1.8057.2.80.46 NAME 'adminOfGroup' DESC 'The id of Group this user or group is administrator of' EQUALITY distinguishedNameMatch SYNTAX 1.3.6.1.4.1.1466.115.121.1.12 )
 olcAttributeTypes: {41}( 1.3.6.1.4.1.8057.2.80.47 NAME 'adminOfFacility' DESC 'The id of Facility this user or group is administrator of' EQUALITY distinguishedNameMatch SYNTAX 1.3.6.1.4.1.1466.115.121.1.12 )
 olcAttributeTypes: {42}( 1.3.6.1.4.1.8057.2.80.48 NAME 'eduPersonEntitlement' DESC 'A URI (either URN or URL) that indicates a set of rights to specific resources' EQUALITY caseExactMatch SYNTAX 1.3.6.1.4.1.1466.115.121.1.15 )
+olcAttributeTypes: {43}( 1.3.6.1.4.1.8057.2.80.49 NAME 'timezone' DESC 'User timezone described by ±[hh] (ISO 8601).' EQUALITY caseExactMatch SYNTAX 1.3.6.1.4.1.1466.115.121.1.15 SINGLE-VALUE )
+olcAttributeTypes: {44}( 1.3.6.1.4.1.8057.2.80.50 NAME 'preferredLanguage' DESC 'Language preferred in communication (notifications).' EQUALITY caseExactMatch SYNTAX 1.3.6.1.4.1.1466.115.121.1.15 SINGLE-VALUE )
+olcAttributeTypes: {45}( 1.3.6.1.4.1.8057.2.80.51 NAME 'phone' DESC 'Users phone number.' EQUALITY caseExactMatch SYNTAX 1.3.6.1.4.1.1466.115.121.1.15{16} )
+olcAttributeTypes: {46}( 1.3.6.1.4.1.8057.2.80.52 NAME 'address' DESC 'Users address.' EQUALITY caseExactMatch SYNTAX 1.3.6.1.4.1.1466.115.121.1.15 )
+olcAttributeTypes: {47}( 1.3.6.1.4.1.8057.2.80.53 NAME 'aups' DESC 'Users approved AUPs.' EQUALITY caseExactMatch SYNTAX 1.3.6.1.4.1.1466.115.121.1.15 )
+olcAttributeTypes: {48}( 1.3.6.1.4.1.8057.2.80.54 NAME 'allowRegistration' DESC 'True, if service (facility) allowes user self-registration.' EQUALITY caseIgnoreMatch SYNTAX 1.3.6.1.4.1.1466.115.121.1.15 SINGLE-VALUE )
+olcAttributeTypes: {49}( 1.3.6.1.4.1.8057.2.80.55 NAME 'registrationURL' DESC 'URL to the registration form for this service (facility).' EQUALITY caseExactMatch SYNTAX 1.3.6.1.4.1.1466.115.121.1.15 SINGLE-VALUE )
+olcAttributeTypes: {50}( 1.3.6.1.4.1.8057.2.80.56 NAME 'dynamicRegistration' DESC 'True if you want that user can select Vo and Group in service (facility) registration process.' EQUALITY caseIgnoreMatch SYNTAX 1.3.6.1.4.1.1466.115.121.1.15 SINGLE-VALUE )
+olcAttributeTypes: {51}( 1.3.6.1.4.1.8057.2.80.57 NAME 'reqAups' DESC 'List of required AUPs, which user has to approve before accessing the service.' EQUALITY caseExactMatch SYNTAX 1.3.6.1.4.1.1466.115.121.1.15 )
```

Comment:
I would preffer to rename to "requiredAups" so the name itself would describe more the content